client/src/components/common/DrawerToggleButton/index.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './style.css';
+
+const DrawerToggleButton = ({ clicked, className }) => {
+  return (
+    <article className={className} onClick={clicked}>
```

Comment:
Read more about the article tag 
[here](https://www.w3schools.com/tags/tag_article.asp)